### ISSUE TITLE
Can't connect after 4.30.10 firmware

### ISSUE THREAD

**Issue by Enixmage** [Maassoft/ColorControl#54]
Title says it all. Also will there be an option for the 120hz dolby option? I hear it's really bad @ 60hz (very framey), but would like an option to do black frame insertion and play around with it. Thanks.

**Comment by Enixmage** [comment]
I'll try removing it from the network, Bluetooth redownloading another fresh copy, and giving it another go. I'll let you know if I can get it working. It detects the correct tv. Just isn't connecting. Is there a way to reset settings? Maybe a reg key. I just want to make sure I'm on default settings and try it fresh.

**Comment by Enixmage** [comment]
After starting over I found a work around for the missing info on the Lg controller tab. Goto properties on the .exe and select high dpi setting, select  high dpi "system" or "system enhanced", it makes it so there's more than just the remote on the page and the correct info is displayed. I had apparently pressed the remote control button and it killed all information on the page except the device and device filter box. 

It connects to the Tv just fine, but the nvidia issues are still present. which is weird. I'll clean the drivers with DDU and try to get it working.

**Comment by Maassoft** [comment]
The Nvidia error could be caused by wrong settings. The app lets you combine settings (like YUV color format with VESA (full) dynamic range), but these are not allowed by the Nvidia driver.
I'm aware that there are issues with the GUI when you have scaling set to over 100%. The properties you've changed on the .exe can also be forced in the .config file. Maybe I will make this default with the next release. Then again, I also want to give the GUI a total makeover in a sense that it not uses WinForms anymore but WPF or maybe UWP, but that'll require a lot of time.

**Comment by Enixmage** [comment]
Yeah, that was it. Idk how or why that happened on the day I upgraded the firmware. I probably messed with it with the new firmware and screwed it up. :) All good thanks for the help. 

The only other option I would want personally would be for the sound. I have a soundbar on the tv and the sync goes out every other time I turn on the tv and I have to set the AV Sync adjustment to +5 and then back to 0 the next time. If I could right-click and do this from my PC it would be way less annoying. Others have this same issue. It's not A huge deal and I know you have other things to do with the program with the new firmware and the new UI. However, if you could put it on the to-do list that would be cool. Thanks again.
